src/main/java/org/trigon/hardware/misc/leds/CANdleLEDStrip.java

```diff
@@ -0,0 +1,219 @@
+package org.trigon.hardware.misc.leds;
+
+import com.ctre.phoenix.led.*;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.util.Color;
+import org.trigon.hardware.RobotHardwareStats;
+
+import java.util.function.Supplier;
+
+/**
+ * A LED strip that is controlled by a CANdle, and uses AddressableLED for simulation.
+ */
+public class CANdleLEDStrip extends LEDStrip {
+    private static CANdle CANDLE;
+    private static int LAST_CREATED_LED_STRIP_ANIMATION_SLOT = 0;
+    private final int animationSlot;
+    private final AddressableLEDStrip simulationLEDStrip;
```

Comment:
should't this be static?

Reply:
no, it's a LEDStrip, each LEDStrip has it's own one